mmv1/templates/terraform/examples/redis_cluster_ha.tf.erb

```diff
@@ -11,6 +11,10 @@ resource "google_redis_cluster" "<%= ctx[:primary_resource_id] %>" {
   depends_on = [
     google_network_connectivity_service_connection_policy.default
   ]
+
+  lifecycle {
+    prevent_destroy = <%= ctx[:vars]['prevent_destroy'] == 'true' %>
```

Comment:
```suggestion
    prevent_destroy = <%= ctx[:vars]['prevent_destroy'] %>
```

Reply:
I suspect we just want this! We already have the value recorded in the vars map.

Reply:
Already have the changed pushed. Sorry about that!